indium-chrome.el

```diff
@@ -121,6 +130,13 @@ Try a maximum of NUM-TRIES."
                                         nil
                                       (indium-chrome--read-tab-data))))))
 
+(defun indium-chrome--connect-to-url (host port url)
```

Comment:
is this one still used?

Reply:
Yeah, I let this one to make it ease to test. Let me know if there is a better way.

Reply:
Later, today, I'll give it a try to get this correctly.